File: Chapter 04/C Code/4_14.c
#include <stdio.h>

int main() {
int n, i, x, flag = 0;
printf("Enter the size of the list: ");
scanf("%d", &n);
int a[n];
printf("Enter the elements of the list:\n");
for (i = 0; i < n; i++) {
scanf("%d", &a[i]);
}
printf("Enter the element to be searched: ");
scanf("%d", &x);
for (i = 0; i < n; i++) {
if (a[i] == x) {
flag = 1;
break;
}
}
if (flag == 1) {
printf("%d is present at position %d in the list.\n", x, i+1);
}
else {
printf("%d is not present in the list.\n", x);
}
return 0;
}

File: Chapter 04/Python code/4_14.py
n=int(input("Enter the size of the list: "))
print("Enter the elements of the list:\n");
a=[]
flag=0
for i in range(0, n):
    el=int(input("Enter :"))
    a.append(el)

x=int(input("Enter the element to be searched: "))

for i in range(0, n):
    if (a[i] == x):
        flag = 1
        break
if (flag == 1):
    print(a[i]," is present at position ",i+1, " in the list")
else:
    print(a[i]," is not present in the list")

